stdlib/universe/mode_test.go

```diff
@@ -0,0 +1,309 @@
+package universe_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/stdlib/universe"
+)
+
+func TestMode_Process(t *testing.T) {
```

Comment:
Can you ensure there's a test case for each of the types that mode can process? It looks like the test case only covers strings right now.